trpc/router/project.ts

```diff
@@ -238,6 +242,9 @@ export const project = router({
                                 },
                             },
                         },
+                        {
+                            OR: [{ deleted: null }, { deleted: false }],
```

Comment:
the same here as with `notArchived` - you can use a variable to clean up these copies.